src/components/MealCostCalculator.vue

```diff
@@ -0,0 +1,304 @@
+<template>
+  <div class = "tableContainer">
+    <h3> {{meal.name}} </h3>
+      <table class ="ingredientTable" border ="1">
```

Comment:
Wrong indentation
Remove `border` attribute
Spacing should be `table class="`

Reply:
✔️